gap/matrix.gi

```diff
@@ -763,6 +767,12 @@ InstallGlobalFunction( FindKernelLowerLeftPGroup,
     return true;
   end );
 
+# computes a straight line program (SLP) for an element <g> of for a p-group
+# in <ri> (that corresponds to a matrix group consisting of lower triangular
+# matrices.
+# The SLP is constructed by using the elementar matrices to cancel out the
+# entries in <g>. The coefficents of the process create the SLP.
+# TODO Max H. wants to rewrite the code
```

Comment:
```suggestion
# TODO Max H. wants to rewrite the code to work row-by-row
# instead of blockwise; that should result in both simple and faster code.
```